### PR TITLE
docs: Add iOS uninstall instructions

### PR DESCRIPTION
**Pull request by Lockl00p**
# Pull Request Template

## Type of change

- [ ] Bug fix
- [ ] Feature
- [X] Documentation update

## Description

*ramble here*

## Checklist

- [X] any anime playing
- [X] bumped version
---
- [X] next, prev and replay work
- [X] `-c` history and continue work
- [X] `-d` downloads work
- [X] `-s` syncplay works
- [X] `-q` quality works
- [X] `-v` vlc works
- [X] `-e` select episode works
- [X] `-S` select index works
- [X] `-r` range selection works
- [X] `--dub` and regular (sub) mode both work
- [X] all providers return links (not necessarily on a single anime, use debug mode to confirm)
---
- [X] `-h` help info is up to date
- [X] Readme is up to date
- [X] Man page is up to date

## Additional Testcases

- The safe bet: One Piece
- Episode 0: Saenai Heroine no Sodatekata ♭
- Unicode: Saenai Heroine no Sodatekata ♭
- Non-whole episodes: Tensei shitara slime datta ken (ep. 24.5, ep. 24.9)

### CODE REVIEW

File: ani-cli [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-version_number="4.3.0"
+version_number="4.3.1"
```
Comment (port19x) [inline ani-cli:3]: Actually, don't change the version number on documentation updates

```suggestion
version_number="4.3.0"
```

**Review by port19x — CHANGES_REQUESTED** [review]

File: ani-cli (hunk 1 above)
Comment (Lockl00p) [reply ani-cli]: Alright! Done!

**Comment by port19x** [comment]
Hmm, this is odd

**Review by port19x — APPROVED** [review]

**Comment by port19x** [comment]
Oh fucking hell.
Only "Validate PR Title" should always be required, the rest only runs if applicable.
I instead told pystardust to make all the checks required, as I assumed that a non-running check would automatically be ignored.

This will have to wait until Saturday, I don't wanna disturb pystardust as he's writing the final pages of his thesis

**Comment by port19x** [comment]
Might make him just disable mandatory checks period

**Comment by port19x** [comment]
> Might make him just disable mandatory checks period

Actually no, this was just a skill issue on my end.
I'll fix the CI configuration for [conditional but required checks](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/troubleshooting-required-status-checks#handling-skipped-but-required-checks)

**Comment by port19x** [comment]
All right, this will work once #1131 is merged and your fork is updated

**Comment by port19x** [comment]
@Lockl00p sync your fork and it should work

**Comment by Lockl00p** [comment]
> @Lockl00p sync your fork and it should work

Done!

**Pull request merged**